modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/src/main/java/constants/__className__WebKeys.java

```diff
@@ -1,10 +0,0 @@
-package ${package}.constants;
```

Comment:
Don't we have code in the ProjectCustomizers and post-generate-groovy that is trying to delete this file?  https://github.com/gamerson/liferay-portal/blob/cd2115c9345de6dfd6228d26bf1d9daae373dd7a/modules/sdk/project-templates/project-templates-npm-angular-portlet/src/main/java/com/liferay/project/templates/npm/angular/portlet/internal/NpmAngularPortletProjectTemplateCustomizer.java#L51

Reply:
I thought these project customizers were for the maven projects. I removed them because the latest samples they submitted no longer use WebKeys